Source/SwiftLintFramework/Extensions/SwiftDeclarationAttributeKind+Swiftlint.swift

```diff
@@ -1,6 +1,15 @@
 import SourceKittenFramework
 
 extension SwiftDeclarationAttributeKind {
+    static var attributesRequiringFoundation: Set<SwiftDeclarationAttributeKind> {
+        return [
+            .objc,
+            .objcName,
+            .objcMembers,
+            .objcNonLazyRealization
```

Comment:
Not in scope for this PR, but I wanted to call it out: the lack of trailing commas on the last line of collection literals made #3040 harder, because I had to special-case a bunch of search and replace. I strongly recommend enforcing trailing commas in any collection-heavy codebase. Just my 2¢